src/akshada_marne/FirstFiveEvenNum.java

```diff
@@ -0,0 +1,40 @@
+package akshada_marne;
+
+public class FirstFiveEvenNum {
+	
+	void loopIf() {
+		for(int num=1; num<=10; num++) {
```

Comment:
You can use parameters here for range. So that you can use this code for multiple ranges instead of start and end